develop/dev-guide-bi-looker-studio.md

```diff
@@ -71,15 +75,56 @@ You can use the S&P 500 dataset provided as an Interactive Tutorial of TiDB Serv
     - Check **Enable SSL**.
         - Click the icon on the right of **MySQL SSL Client Configuration Files**. Then, select the downloaded CA file in step 3. 
 
-    ![Looker Studio: configure connection settings for TiDB Serverless](/media/develop/looker-studio-configure-connection.png)
+    ![Looker Studio: configure connection settings for TiDB cloud](/media/develop/looker-studio-configure-connection.png)
 
 7. Click **AUTHENTICATE**. 
 
 If the authentication succeeds, you can see tables in the database. 
 
+</div>
+<div label="TiDB Dedicated">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **...** menu in the upper-right corner. Then, select **Security Settings**. Settings dialog shows up. 
+
+3. Click **Generate** to generate root password and **Copy** it to the clipboard.
+
+    > **Tip:**
+    >
+    > If you have created a password before, use the original password or click **Regenerate** to generate a new one.
+
+4. Click **Apply** to close the dialog.
+   
+5. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+
+6. Click **Allow Access from Anywhere(Insecure)**.
```

Comment:
not sure from anywhere is a good option. https://support.google.com/looker-studio/answer/7088031?hl=en# seems there is a fixed outbound ip range.

Reply:
Thanks, I checked this IP range, and it works. Changed as suggested.